cfme/tests/automate/test_method.py

```diff
@@ -33,6 +33,8 @@ def test_method_crud(klass):
         initialEstimate: 1/16h
         tags: automate
     """
+    # ToDo: Update this test case for other types of automate methods like builtin, expression, uri,
```

Comment:
Required: Please write #TODO as some effort was made in #8588

Reply:
Thanks :)